eth/api_tracer.go

```diff
@@ -625,6 +619,15 @@ func (api *PrivateDebugAPI) standardTraceBlockToFile(ctx context.Context, block
 	return dumps, nil
 }
 
+func containsTx(block *types.Block, txHash common.Hash) bool {
```

Comment:
Please document this method. I know it's only a package-internal method, but everything else is nicely documented and I'd like to keep it so.